File: android/src/main/java/com/getcapacitor/community/classes/results/IsInitializedResult.java
package com.getcapacitor.community.classes.results;

import com.getcapacitor.JSObject;
import com.getcapacitor.community.interfaces.Result;

public class IsInitializedResult implements Result {

    private final Boolean isInitialized;

    public IsInitializedResult(Boolean isInitialized) {
        this.isInitialized = isInitialized;
    }

    @Override
    public JSObject toJSObject() {
        JSObject result = new JSObject();

        if (isInitialized != null) result.put("isInitialized", isInitialized);

        return result;
    }
}

File: ios/Sources/BridgefyPlugin/Classes/Results/IsInitializedResult.swift
import Foundation
import Capacitor

@objc public class IsInitializedResult: NSObject, Result {
    let isInitialized: Bool

    init(_ isInitialized: Bool) {
        self.isInitialized = isInitialized
    }

    public func toJSObject() -> AnyObject {
        var result = JSObject()

        result["isInitialized"] = isInitialized

        return result as AnyObject
    }
}
